Tests/Core/AWSClientRuntimeTests/Sigv4/SigV4SigningTests.swift

```diff
@@ -17,6 +18,275 @@ class Sigv4SigningTests: XCTestCase {
         CommonRuntimeKit.initialize()
     }
 
+    // Test success case
+    func testSignRequestSuccess() async throws {
+        let dateString = "2024-01-16T12:36:00Z"
+        guard let date = TimestampFormatter(format: .dateTime).date(from: dateString) else {
```

Comment:
nit: For brevity in a test case, could just `try XCTUnwrap()` the date here.

Reply:
As suggested, changed all test cases in this test class to use XCTUnwrap where applicable.